spec/services/creators/gross_income_eligibility_creator_spec.rb

```diff
@@ -3,38 +3,37 @@
 module Creators
   RSpec.describe GrossIncomeEligibilityCreator do
     let(:summary) { assessment.applicant_gross_income_summary }
+    let(:assessment) { create :assessment, :with_gross_income_summary, proceedings: [%w[DA002 A], %w[SE013 Z]] }
+    let(:proceeding_types) { assessment.proceeding_types }
+    let(:submission_date) { assessment.submission_date.to_date }
 
-    around do |example|
-      travel_to Date.new(2021, 4, 20)
-      example.run
-      travel_back
+    before do
+      ::Utilities::ProceedingTypeThresholdPopulator.call(assessment)
     end
 
     subject(:creator) do
       described_class.call(dependants:,
-                           proceeding_types: assessment.proceeding_types,
-                           submission_date: assessment.submission_date, total_gross_income: 0)
+                           proceeding_types:,
+                           submission_date:, total_gross_income: 0)
     end
 
-    context "version 6" do
-      let(:assessment) { create :assessment, :with_gross_income_summary, proceedings: [%w[DA002 A], %w[SE013 Z]] }
-      let(:proceeding_types) { assessment.proceeding_types }
-      let(:dependants) { [] }
-
-      it "creates a capital eligibility record for each proceeding type" do
-        expect(creator.map(&:proceeding_type)).to match_array(proceeding_types)
-      end
-
-      it "creates eligibility record with correct waived thresholds" do
-        pt = proceeding_types.find_by!(ccms_code: "DA002", client_involvement_type: "A")
-        elig = creator.detect { _1.proceeding_type.ccms_code == "DA002" }
-        expect(elig.upper_threshold).to eq pt.gross_income_upper_threshold
-        expect(elig.lower_threshold).to be_nil
+    context "without MTR" do
+      around do |example|
+        travel_to Date.new(2021, 4, 20)
+        example.run
+        travel_back
       end
 
       context "no dependants" do
         let(:dependants) { [] }
 
+        it "creates eligibility record with correct waived thresholds" do
+          pt = proceeding_types.find_by!(ccms_code: "DA002", client_involvement_type: "A")
+          elig = creator.detect { _1.proceeding_type.ccms_code == "DA002" }
+          expect(elig.upper_threshold).to eq pt.gross_income_upper_threshold
+          expect(elig.lower_threshold).to be_nil
+        end
+
         it "creates eligibility record with correct un-waived thresholds" do
           pt = proceeding_types.find_by!(ccms_code: "SE013", client_involvement_type: "Z")
```

Comment:
@starswan We can move these 2 redundant lines on top of the context
Eg:
      let(:pt) { proceeding_types.find_by!(ccms_code: "SE013", client_involvement_type: "Z") }
      let(:elig) { creator.detect { _1.proceeding_type.ccms_code == "SE013" } }